packages/rainbowkit/src/wallets/Wallet.ts

```diff
@@ -40,9 +36,7 @@ export type Wallet<C extends Connector = Connector> = {
     browserExtension?: string;
     qrCode?: string;
   };
-  createConnector: (connectorArgs: {
-    chainId?: number;
-  }) => RainbowKitConnector<C>;
+  createConnector: () => RainbowKitConnector<C>;
```

Comment:
idk if we should convert this from a function to a plain rainbowkit connector?

Reply:
🔥 

> idk if we should convert this from a function to a plain rainbowkit connector now that there are no params?

not sure, is a plain rainbowkit connector a simpler api?

---

just a reminder to update [docs props table](https://rainbowkit.vercel.app/docs/custom-wallets) when we settle on the api

<img width="547" alt="CleanShot 2022-06-06 at 18 05 50@2x" src="https://user-images.githubusercontent.com/372831/172199760-7e02c0e0-c4df-450d-8b40-21f063a6d506.png">

Reply:
Yeah it will be a simpler API! If we do go in this direction though, I wouldn't be sure on what to rename the property since there is already a `connector` property in the `RainbowKitConnector` type:

```diff
type RainbowKitConnector<C extends Connector = Connector> = {
  connector: C;
  ...
  };
};

export type Wallet<C extends Connector = Connector> = {
  ...
- createConnector: (connectorArgs: {
-   chainId?: number;
-  }) => RainbowKitConnector<C>;
+ connector: RainbowKitConnector<C>
};
```

Would having 2 `connector`s be weird?